datasheet/Zubax_GNSS2_Datasheet.tex

```diff
@@ -0,0 +1,777 @@
+\end{ZubaxSimpleTable}
+
+\section{Via USB/UART}
+Once started, the bootloader exposes a CLI via either DCD port or USB. The USB is always preferred if it is connected to the host; otherwise the CLI falls back to the UART interface of the DCD port.
+
+The CLI can be used to query the state of the bootloader, modify it, obtain the information about the running firmware, and upgrade it if necessary.
+
+The CLI prompt is of the following format: \textbf{StateName>}, which features the human readable name of the current state of the bootloader, followed by the ASCII greater character (ASCII code 62), followed by a space. For example:  \textbf{BootDelay>}. Such prompt allows the user (or software) to easily identify the state of the bootloader.
+\subsection{CLI commands}
+\textbf{reboot}\\
+Restarts the bootloader.
+
+\textbf{zubax{\_}id}\\
+This is the standard command supported by all products by Zubax Robotics that have a CLI. It takes no arguments, and outputs a YAML key-value dictionary containing the vital information about the device, the firmware it is running, unique ID, installed certificates of authenticity, version of the hardware, and possibly some other information. Aside from the standard fields, this command also provides at least the following fields in its output:
+
+\begin{itemize}
+\item \textbf{bl{\_}version} - bootloader version, major and minor.
+\item \textbf{bl{\_}vcs{\_}commit} - build identifier of the bootloader.
+\item \textbf{mode} - set to the string \textbf{bootloader} to indicate that the bootloader is running.
+\end{itemize}
+If there is a valid firmware, its version information will also be provided via the standard fields \textbf{fw{\_}version} and  \textbf{fw{\_}vcs{\_}commit}. If the bootloader could not find a valid firmware, these fields will be omitted.
+
+\textbf{wait}\\
+Do not boot the application. If the current state is \textbf{BootDelay}, the state will be switched to \textbf{BootCancelled}. In all other states the command will have no effect.
+
+\textbf{download}\\
+Start the serial receiver and prepare to receive the new firmware image as a flat binary via the serial link using either YMODEM, XMODEM, or XMODEM-1K. The bootloader will automatically detect which protocol to use. According to the YMODEM specification, if no transfer was initiated by the host within one minute, the command will exit with an error. Possible error codes are defined in the table above.
+
+Note that while this command is running, the CLI will be unavailable, since the same serial link will be temporarily occupied by the file transfer protocol.
+
+If the YMODEM protocol is used, the file name field in the transfer header packet will be ignored.
+
+There are heaps of software products and scripts that support these file transfer protocols. For instance, the popular program \textbf{sz} (available on most Linux distributions) can be used as follows:\\
+\texttt{sz -vv --ymodem --1k \$file > \$port < \$port}
```

Comment:
This should be a code block (minted) with syntax highlighting set for Bash, would you agree? Also, I see that you're at it again with the forced newlines. Ugh.